traffic_ops/app/lib/UI/ConfigFiles.pm

```diff
@@ -852,18 +852,47 @@ sub hosting_dot_config {
 	return $text;
 }
 
-sub storage_dot_config_volume_text {
+sub storage_dot_config_letter_to_id_mapping($$@) {
+	my $letters      = shift;        
+	my $hash_key_ids = shift;        
+	my @postfix      = @_;
+	my %mapping;                                            
+                                                            
+	if (defined($hash_key_ids)) {                           
+		my @tmp_list = split( /,/, $hash_key_ids);          
+                                                            
+		# find the minimum of the two list sizes
```

Comment:
none of that should be in there, actually.  let me clean that up.  must have had some leakage from another branch.